shell/platform/windows/angle_surface_manager.h

```diff
@@ -37,6 +37,9 @@ class AngleSurfaceManager {
   // Target represents the visual entity to bind to.  Width and
   // height represent dimensions surface is created at.
   bool CreateSurface(WindowsRenderTarget* render_target,
+#ifndef WINUWP
```

Comment:
Not sure how much of the angle code will need to be conditionalized vs a case statement.  Maybe the entrypoint won't but the internals will, that will become obvious as you work through it.